client/src/services/follows.service.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+import { config } from "../config/config";
+
+const loadFollowers = (email, pageNumber) => {
+    return axios.get(`${config.serverUrl}/follows/${email}/followers/`, {
+        params: { pageNumber }
+    }).then(response => response.data)
+        .catch(error => {
+            console.error(`Error while trying to fetch followers details. Error: ${error}`);
+            throw error;
+        });
+};
+
+const loadFollowings = (email, pageNumber) => {
+    return axios.get(`${config.serverUrl}/follows/${email}/following/`, {
+        params: { pageNumber }
+    }).then(response => response.data)
+        .catch(error => {
+            console.error(`Error while trying to fetch most followings details. Error: ${error}`);
+            throw error;
+        });
+};
+
+const updateFollower = (action, emailFrom, emailTo) => {
+    return axios.post(`${config.serverUrl}/follows`, { action, emailFrom, emailTo })
```

Comment:
This method will work for both add and remove. Update `console.error` so it won't relate to "remove" only.